client/components/date-range/test/index.js

```diff
@@ -63,22 +57,26 @@ describe( 'DateRange', () => {
 				/>
 			);
 
-			const actualStartDate = wrapper.state().startDate;
-			const actualEndDate = wrapper.state().endDate;
+			const [ actualStartDate, actualEndDate ] = screen
+				.getByLabelText( 'Select date range' )
+				.querySelector( 'span' )
```

Comment:
This feels a bit fragile; we could have a bunch of `span`s inside the component, and we could potentially grab a wrong one here. Any better way to access the range wrapper?

Reply:
As this was one of the very first examples I worked on, it's very possible there might be already a better way, or if not, I'll see what we can change in the component to make it less fragile. Thanks for the nudge :)

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/63879/commits/addf61eb1ecdcfb088f502364eb0492d5861ef03, and abstracted into a small helper in https://github.com/Automattic/wp-calypso/pull/63879/commits/02bef0c96e0d56c22b21b44c47e7863bb078abdc.